api/src/main/resources/liquibase.xml

```diff
@@ -316,17 +316,21 @@
     <changeSet id="create-auto-close-queue-entry-task-20230616" author="makombe">
         <preConditions>
             <sqlCheck expectedResult="0">
-                SELECT COUNT(*) FROM scheduler_task_config WHERE name = "Auto close queue entries Task";
+                SELECT COUNT(*) FROM scheduler_task_config WHERE name = 'Auto close queue entries Task';
             </sqlCheck>
         </preConditions>
-        <sql>
-            INSERT INTO
-            scheduler_task_config(name, description, schedulable_class, start_time, start_time_pattern, repeat_interval,
-            start_on_startup, created_by, date_created, uuid)
-            VALUES
-            ('Auto close queue entries Task', 'Auto close queue entries Task', 'org.openmrs.module.queue.tasks.AutoCloseQueueEntryTask',
-            TIMESTAMP(CURDATE()), 'MM/dd/yyyy HH:mm:ss', 86400, TRUE, 1, NOW(), UUID());
-        </sql>
+        <insert tableName="scheduler_task_config">
```

Comment:
Did you confirm that that modifying this changeset will not lead to validation checksum errors?

Reply:
should I create a separate changeset for Postgresql?

Reply:
@dkayiwa updated this by adding a validCheckSum tag.